pkgs/development/ocaml-modules/apron/default.nix

```diff
@@ -13,8 +13,21 @@ stdenv.mkDerivation rec {
   buildInputs = [ perl gmp mpfr ppl ocaml findlib camlidl ];
   propagatedBuildInputs = [ mlgmpidl ];
 
-  prefixKey = "-prefix ";
-  preBuild = "mkdir -p $out/lib/ocaml/${ocaml.version}/site-lib/stublibs";
+  outputs = [ "out" "bin" "dev" ];
+
+  configurePhase = ''
+    runHook preConfigure
+    ./configure -prefix $out
+    mkdir -p $out/lib/ocaml/${ocaml.version}/site-lib/stublibs
+    runHook postConfigure
+  '';
+
+  postInstall = ''
+    mkdir -p $dev/lib
+    mv $out/lib/ocaml $dev/lib/
```

Comment:
Can we use install here?

Reply:
What do you mean?

Reply:
``install -Dm555 $out/lib/ocaml $dev/lin

Reply:
But `install` does not remove the source, does it?

Reply:
No, it does not. I this required here?

Reply:
Yes. It is the very purpose of this PR.